plugin/src/test/java/io/jenkins/plugins/casc/impl/configurators/DataBoundConfiguratorTest.java

```diff
@@ -90,6 +88,18 @@ public void export_mapping_should_not_be_null() throws Exception {
         assertEquals("plainText", configNode.getScalarValue("markupFormatter"));
     }
 
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Test
+    public void shouldThrowConfiguratorException() throws ConfiguratorException{
+        expectedException.expect(ConfiguratorException.class);
+        expectedException.expectMessage("Message assertion");
+
+        //Need a way to throw the exception here using the same databound config params.
+        throw new ConfiguratorException();
```

Comment:
Need a way to add custom params to this exception to test it out.

Reply:
You can create a dummy data-bound object or use `Foo` to make it fail 😅

Reply:
Yeah either that or just construct a custom Configuration exception.

Reply:
I'd prefer we test the real thing 😅

Reply:
Error:(22, 41) java: package io.jenkins.plugins.casc.snakeyaml does not exist
Yeah could you just help me with this error.It throws up when I try to run the individual test

Reply:
`mvn install -DskipTests` on the root

Reply:
@sladyn98 what @casz meant by dummy data-bound object was to implement something like this:
https://github.com/jenkinsci/configuration-as-code-plugin/blob/62764fa032dd7505445fcf15335a78ad60598ce6/plugin/src/test/java/io/jenkins/plugins/casc/impl/configurators/DataBoundConfiguratorTest.java#L93

and then pass invalid arguments and assert the message is what you expect